backend/src/main/java/com/dhbw/tutorsystem/tutorial/TutorialController.java

```diff
@@ -505,7 +582,7 @@
                     "tutorialStart", tutorial.getStart(),
                     "tutorialEnd", tutorial.getEnd(),
                     "tutorialDurationMinutes", tutorial.getDurationMinutes(),
-                    "reason", reason);
+                    "reason", reason != null? reason : "");
```

Comment:
Kommentar, dass klar wird dass man das nur macht um die NotNull-Bedingung der Map zu umgehen (gilt nur für Map.of Ergebnisse)